Web/Edubase.Web.UI/Areas/Establishments/Views/EstablishmentsSearch/Downloads/SelectDataset.cshtml

```diff
@@ -24,7 +24,7 @@
             <div class="govuk-grid-column-full">
                 <div class="govuk-breadcrumbs">
                     <ol class="govuk-breadcrumbs__list">
-                        <li class="govuk-breadcrumbs__list-item">@Html.ActionLink("Search", "Index", "Search", new { area = "" }, new { @class = "govuk-breadcrumbs__link" })</li>
+                        <li class="govuk-breadcrumbs__list-item">@Html.ActionLink("Search", "Index", "Search", new { area = "", SelectedTab = "Establishments" }, new { @class = "govuk-breadcrumbs__link" })</li>
```

Comment:
SelectedTab should possibly use the Model.SearchSource (referenced in line 17) in a similar manner to that used when building the link on EstablishmentSearch/Index.cshtml

hard-coding might be valid if this is only ever going to be driven by the establishment search, in which case it might be worth using the SearchViewModel.Tab.Establishments enum ref rather than a string as ensures consistency.